code-samples/creating-your-bot/command-handling/deploy-commands.js

```diff
@@ -1,10 +1,12 @@
-const fs = require('node:fs');
+const path = require('path');
+const fs = require('fs');
```

Comment:
keep fs in the node scope, please.

Reply:
may I convert modules to ESM?

Reply:
The rest of the guide uses cjs based code, so should this section